pkg/ssh/shell/runLoadTest.sh

```diff
@@ -3,4 +3,5 @@
 if [ -e {{ .LoadTestResultFile }} ]; then
   rm {{ .LoadTestResultFile }}
 fi
-eval {{ .LoadTestCommand }} > {{ .LoadTestResultFile }}
+eval {{ .LoadTestCommand }} > {{ .LoadTestResultFile }} 2> /dev/null &
```

Comment:
I also suggest we use `nohup` instead unless and don't call it directly in the loadtest command